manual/moderncv_userguide.tex

```diff
@@ -1,462 +1,996 @@
+  \item \lstinline!\githubsocialsymbol!
+  \item \lstinline!\gitlabsocialsymbol!
+  \item \lstinline!\stackoverflowsocialsymbol!
+  \item \lstinline!\googlescholarsocialsymbol!
+  \item \lstinline!\telegramsocialsymbol!
+  \item \lstinline!\whatsappsocialsymbol!
+  \item \lstinline!\signalsocialsymbol!
+  \item \lstinline!\matrixsocialsymbol!
+  \item \lstinline!\orcidsocialsymbol!
+  \item \lstinline!\researchgatesocialsymbol!
+  \item \lstinline!\researcheridsocialsymbol!
+  \item \lstinline!\bitbucketsocialsymbol!
+  \item \lstinline!\skypesocialsymbol!
+  \item \lstinline!\bornsymbol!
+\end{itemize}
+The possible options for \Code{<\\symbol>} depend on the package that is used. 
+By default the \Code{marvosym} package is loaded if \Code{pdflatex} is used and the packages 
+\code{academicons} and \Code{fontawesome5} are loaded if \Code{lualatex} or \Code{xelatex} are 
+employed. The documentation including full lists of all available symbols and icons can be found 
+in the respective package documentations \medskip
+
+\begin{tabular}{l}
+  \url{https://ctan.org/pkg/marvosym}\\[1ex]
+  \url{https://ctan.org/pkg/fontawesome5}\\[1ex]
+  \url{https://ctan.org/pkg/academicons}
+\end{tabular}
+
+\paragraph{Example.}
+If one wanted to use the dingbat fonts to substitute fixed phone symbol one would load the \Code{pifont} package in the preamble, then substitute the default symbol \Code{\\fixedphonesymbol} with the dingbat symbol \ding{38}\ by:
+\begin{lstlisting}
+  \renewcommand*{\fixedphonesymbol}{\ding{38}~}
+\end{lstlisting} 
+
+\subsubsection{Listing labels}
+The labels used in \Code{itemize} environments or the \Moderncv macros \code{cvlistitem} and \Code{cvlistdoubleitem} can be affected in two different ways:
 
-\begin{figure}[!ht]
- \centering
- \fbox{\includegraphics[angle=90,width=\textwidth]{test_cvsection2.pdf}}\\
- \fbox{\includegraphics[angle=90,width=\textwidth]{test_cvsection1.pdf}} 
-\caption{Example of usage of items inside cvsections.}
- \label{fig:sections}
-\end{figure}
+\begin{itemize}
+ \item By redefining the internal commands \code{\\labelitemi}, \code{\\labelitemii} as well as
+  \code{\\labelitemiii} and \code{\\labelitemiv}, e.g. 
+  \begin{lstlisting}
+  \renewcommand{\labelitemi}{-}.
+  \end{lstlisting}
+  This will affect both, the \Code{itemize} environments as well as the macros \Code{cvlistitem} and \Code{cvlistdoubleitem}.
+  \item If one only whishes to change the labels of the macros \Code{cvlistitem} and \Code{cvlistdoubleitem}, one needs to redefine the \code{\\listitemsymbol}, e.g.
+\begin{lstlisting}
+  \renewcommand{\listitemsymbol}{-}.
+\end{lstlisting}
+This will leave the definitions of \Code{itemize} environments untouched.
+\end{itemize}
 
-\section{More customizations}
+\subsection{Adjusting lengths}
+\todo{Add more adjustable lengths}
+Some of the lengths in moderncv can be adjusted. 
+The hints column can be adjusted by issuing
+\begin{lstlisting}
+  \setlength{\hintscolumnwidth}{3cm}
```

Comment:
showing the default for each length in this section (as you do for `\cvsectionstretchability`) would be beneficial